torch/_inductor/utils.py

```diff
@@ -1069,8 +1072,8 @@ def use_cpp_packed_gemm_template(layout, mat1, mat2):
         m,
         n,
         k,
-        input_dtype=layout.dtype,
-        output_dtype=torch.float,
+        input_dtype=mat1.get_dtype() if int8_gemm else layout.dtype,
```

Comment:
```suggestion
        input_dtype=mat1.get_dtype(),
```

Reply:
Thanks for the comment and changed accordingly.